src/Console/ConfigurationResolver.php

```diff
@@ -314,13 +314,15 @@ private function computeConfigFiles()
         }
 
         $path = $this->path;
-        if (is_file($path) && $dirName = pathinfo($path, PATHINFO_DIRNAME)) {
-            $configDir = $dirName;
-        } elseif ($this->isStdIn || null === $path) {
+
+        if ($this->isStdIn || 0 === count($path)) {
             $configDir = $this->cwd;
-            // path is directory
+        } elseif (1 < count($path)) {
+            throw new InvalidConfigurationException('For multiple paths config parameter is required.');
```

Comment:
Thanks for the answer! That's not really in my interest ;) but at least I understand now.

To give some context I'm working on https://github.com/mnapoli/sniff What I want to do is basically the same thing as `.php_cs` except with a JSON file. My current approach is maybe not the best but at least I made it work, I run this:

```
php-cs-fixer path1 path2 … --config=null-config-file.php --rules=...
```

And here is the "null config file": https://github.com/mnapoli/sniff/blob/master/php-cs-null-config.php